### PR TITLE
Document how to handle network issue with keadm

### LINKED ISSUE #1967: A solution for network error when executing "keadm init"
**What would you like to be added/modified**:
Append a command parameter `--down-source` to specify an available download source when executing "keadm init".
I means changing `KubeEdgeCRDDownloadURL` to a variable.

**Why is this needed**:
When executing "keadm init",an error occurs due to network error.

F0724 10:07:36.343995    9709 keadm.go:27] failed to run 'sh -c cd /etc/kubeedge/crds/devices && wget -k --no-check-certificate --progress=bar:force https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha1_device.yaml' because of error : exit status 4

And when I set $http_proxy$, "keadm init" didn't report error.

#1930 has the similar problem.

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

Document how to handle network issue with `keadm`.

**What type of PR is this?**
/kind documentation

**Which issue(s) this PR fixes**:

Fixes #1967 #1930

### CODE REVIEW

**Comment by ttlv** [comment]
it seems that the doc is about how to install ke node manually

**Comment by daixiang0** [comment]
ping @fisherxu

File: docs/setup/keadm.md [hunk 1]
```diff
@@ -9,6 +9,35 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 - Currently support of `keadm` is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
 - Need super user rights (or root rights) to run.
 
+**Note**
```
Comment (fisherxu) [inline docs/setup/keadm.md:12]: We should avoid add too many info in setup docs, can move it to troubleshooting

Comment (daixiang0) [reply docs/setup/keadm.md:12]: Done

**Pull request merged**